apps/contestedroll.js

```diff
@@ -543,7 +554,7 @@ export class ContestedRoll {
             message.update({ flags: { 'monks-tokenbar': flags } });
     }
 
-    static async onRollAll(message, e) {
+    static async onRollAll(all, message, e) {

```

Comment:
Could you change this to `tokentype` just to be consistent?
It makes more sense when using it with SavingThrow, but I like the consistency.